venus/src/pages/spot/components/photo-gallery/Photo.tsx

```diff
@@ -1,15 +1,20 @@
 import LoadingSpinner from "../../../../components/loading-spinner/LoadingSpinner.jsx";
-import { useState } from "react";
+import React, { useState } from "react";
+import Img from "../../../../model/interface/img";
 
-export default function Photo({ photo, onClick = () => {}, ...props }) {
-  const [isLoading, setIsLoading] = useState(true);
+type PhotoProps = {
+  photo: Img;
+} & React.HTMLAttributes<HTMLDivElement>;
```

Comment:
```suggestion
type PhotoProps = {
  photo: Img;
} & HTMLAttributes<HTMLDivElement>;
```
Wystarczy samo to React nie jest tu potrzebne

Reply:
Zmieniłem